File: src/main/java/itmo/is/project/mapper/module/DockModuleBlueprintMapper.java
package itmo.is.project.mapper.module;

import itmo.is.project.dto.module.dock.DockModuleBlueprintDto;
import itmo.is.project.mapper.EntityMapper;
import itmo.is.project.model.module.dock.DockModuleBlueprint;
import org.mapstruct.Mapper;

@Mapper(componentModel = "spring")
public interface DockModuleBlueprintMapper extends EntityMapper<DockModuleBlueprintDto, DockModuleBlueprint> {
}

File: src/main/kotlin/itmo/isproject/mapper/module/dock/DockModuleBlueprintMapper.kt
package itmo.isproject.mapper.module.dock

import itmo.isproject.dto.module.dock.DockModuleBlueprintDto
import itmo.isproject.mapper.EntityMapper
import itmo.isproject.model.module.dock.DockModuleBlueprint
import org.mapstruct.Mapper

@Mapper
interface DockModuleBlueprintMapper : EntityMapper<DockModuleBlueprintDto, DockModuleBlueprint>
